### ISSUE TITLE
Use python-libstorj bindings in EasyStorj GUI

### ISSUE THREAD

**Issue by lakewik** [lakewik/storj-gui-client#104]
I started to integrate python-libstorj bindings to EasyStorj. EasyStorj will have two upload and download modes: native python and binding mode because libstorj-c have some adventages like Reed-Solomon  erasure encoding but also some disadventages like unability to set max shard size, retry count, informations about download/upload progress for each shard, concurrency - but i going to also add these functionalities to libstorj-c in future,

**Comment by marcorosa** [comment]
Good! I guess we should do this in another branch. Moreover, please note that actual version of uploading and downloading of the python-bindings are not working (they're still WIP)

**Comment by lakewik** [comment]
Yes, i tested it and we will start implementing working features like buckets and file listing :)
